daemon/protocol/python/tests/daemon_cli_test.py

```diff
@@ -5,140 +5,180 @@
 import shutil
 import subprocess
 import tempfile
-import time
-import unittest
 
 
+gopath = re.split(":", os.environ['GOPATH'])[0]
```

Comment:
I don't fully understand what was going on... was getting file not found error - I think wrt to finding the /socket file - I'm guessing it has something to do with permissions or working dirs being different if running from python's test framework.